src/liballoc/string.rs

```diff
@@ -2179,6 +2179,14 @@ impl From<&str> for String {
     }
 }
 
+#[unstable(feature="from_ref_string", issue="59827")]
```

Comment:
new impls are insta-stable, unfortunately.

Reply:
Ah, OK. What version should I put then?

Reply:
1.36.0. I don't think this PR could be merged in a day (before the coming release )

Reply:
.34 would be ambitious yes; I guessed .35, but .36 is fine too.